### ISSUE TITLE
CKEditor insisting on converting line breaks into paragraphs

### ISSUE THREAD

**Issue by Graham-72** [backdrop/backdrop-issues#1350]
How can I stop CKEditor from adding empty paragraphs between images? The filter  'Convert line breaks into HTML ' is not ticked but still it does this, if not when closing the edit frame it does it when re-opening.

![edit basic page club officers - deben rowing club 2015-11-13 17-43-12](https://cloud.githubusercontent.com/assets/10559274/11153048/33c76474-8a2e-11e5-832a-80be06357000.png)

**Comment by Graham-72** [comment]
I think this must be a fault and if so I would like it to be added to meta issue #1087

**Comment by klonos** [comment]
I just realized that you might not have permissions to edit the list in the meta-issue. Added the issues you reported there for you (and did some clean-up while at it).

**Comment by Graham-72** [comment]
Thanks @klonos , :+1:  how would we manage without you?

**Comment by klonos** [comment]
:smile:

**Comment by quicksketch** [comment]
Thanks @Graham-72. Indeed looks like a bug either in CKEditor itself or in our Image plugin.

**Comment by quicksketch** [comment]
Hm, I couldn't reproduce this problem in latest Firefox. When the "Convert line breaks into HTML" is unticked or not, I get the same result.

When entering two images one right after the other I get this markup:
```
<p><img alt="" data-file-id="14" src="http://backdrop.dev/files/inline-images/selection_027.png" height="157" width="688"><img alt="" data-file-id="17" src="http://backdrop.dev/files/inline-images/Selection_001.jpg" height="420" width="720"></p>
```

When inserting with a return between the two images I get:
```
<p><img alt="" data-file-id="14" src="http://backdrop.dev/files/inline-images/selection_027.png" height="157" width="688"></p>

<p><img alt="" data-file-id="17" src="http://backdrop.dev/files/inline-images/Selection_001.jpg" height="420" width="720"></p>
```

Note that if you want a smaller gap in-between two vertical images, you can also use `Shift` + `Enter` in CKEditor to create a `<br>` tag instead. However... `<br>` is not in the default tag list (maybe something we should change?) So it has to be added in `admin/config/content/formats/filtered_html`.

**Comment by Graham-72** [comment]
Yes, I confirm that I am no longer experiencing this problem.
Thanks for checking.

**Comment by quicksketch** [comment]
Yay, great. Must have been fixed in 1.4.0 when we updated CKEditor.

**Comment by klonos** [comment]
👍 another one down 😉
